test/examples/ucp_client_server.c

```diff
@@ -236,18 +233,26 @@ static ucs_status_t request_wait(ucp_worker_h ucp_worker, test_req_t *request)
 }
 
 static int request_finalize(ucp_worker_h ucp_worker, test_req_t *request,
-                            int is_server, char *recv_message)
+                            int is_server, char *recv_message, int current_iter)
 {
     ucs_status_t status;
     int ret = 0;
 
     status = request_wait(ucp_worker, request);
-    if (status != UCS_OK){
+    if (status != UCS_OK) {
         fprintf(stderr, "unable to %s UCX message (%s)\n",
                 is_server ? "receive": "send", ucs_status_string(status));
-        ret = -1;
-    } else {
-        print_result(is_server, recv_message);
+        return -1;
+    }
+
+    if (is_server) {
+        server_iters_cnt++;
```

Comment:
this should be a local variable

Reply:
but it is used both from here and from the server function

Reply:
it can be a parameter to this function as what it should print